javers-spring/src/main/java/org/javers/spring/auditable/aspect/springdata/JaversSpringDataAuditableRepositoryAspect.java

```diff
@@ -17,6 +18,7 @@
  * Calls {@link Javers#commitShallowDelete(String, Object, Map)} on arguments passed to delete() methods.
  */
 @Aspect
+@Order(0)
```

Comment:
Adding @Order(0) so that any other advice on the same point cut expression can be evaluated before this.